runtime/src/ancient_append_vecs.rs

```diff
@@ -704,7 +706,20 @@ pub mod tests {
                 .iter()
                 .map(|meta| meta.pubkey())
                 .collect::<Vec<_>>(),
-            vec![&pk2]
+            vec![&pk_1_ref]
+        );
+        assert_eq!(
+            accounts_to_combine
```

Comment:
Would be a bit more concise if we store `accounts_to_combine.accounts_to_combine.first().unwrap().alive_accounts.one_ref.accounts` in a temporary